SAMtags.tex

```diff
@@ -235,12 +249,30 @@ \subsection{Barcodes}
 \begin{description}
 \item[BC:Z:\tagvalue{sequence}]
 Barcode sequence, with any quality scores stored in the {\tt QT} tag.
+The {\tt BC} tag should match the {\tt QT} tag in length. 
+What one does with multiple sample barcodes (e.g. one on each end of the template) is left to the implementation. 
+One current implementation concatenates all the barcodes and qualities and places a dash (`{\tt -}') between the barcodes from the same template. 
+
+\item[QT:Z:\tagvalue{qualities}] 
+Phred quality of the sample-barcode sequence in the {\tt BC} (or {\tt RT}) tag. 
+Same encoding as {\sf QUAL}. 
+See {\tt BC} for more detail.
 
-\item[QT:Z:\tagvalue{qualities}]
-Phred quality of the barcode sequence in the {\tt BC} (or {\tt RT}) tag. Same encoding as {\sf QUAL}.
+\item[QX:Z:\tagvalue{qualities}] 
+Phred quality of the molecular-barcode sequence in the {\tt RX} tag. 
+Same encoding as {\sf QUAL}. 
+
+The {\tt RX} tag should match the {\tt QX} tag in length. 
+What one does with multiple molecular barcodes (e.g., one on each end of the template) is left to the implementation. 
+One current implementation concatenates all the barcodes and qualities with no separator between the different barcodes.
 
 \item[RT:Z:\tagvalue{sequence}]
 Deprecated alternative to {\tt BC} tag originally used at Sanger.
+
+\item[RX:Z:\tagvalue{sequence}] 
+Raw (uncorrected) molecular-barcode bases, with any quality scores stored in the {\tt QX} tag. 
```

Comment:
I don't see a lot of value in limiting `RX` to be the "raw (uncorrected)" bases.  Or at least, if so, then we should define the tag for the corrected bases.  I have pipelines that go from raw -> corrected -> source molecule assignment, and each is a different logical step.